src/Uno.UI.RemoteControl.Host/Program.cs

```diff
@@ -61,6 +62,11 @@ static async Task Main(string[] args)
 				throw new ArgumentException($"The httpPort parameter is required.");
 			}
 
```

Comment:
unit tests? when the solution is null, outputs correct messages for missing or invalid inputs, initialization of the LoggerFactory, for example

Reply:
We don't have testing yet for dev-server.